Plandomizer.py

```diff
@@ -921,7 +921,12 @@ def configure_gossip(self, spoiler, stoneIDs):
             matcher = self.pattern_matcher(name)
             stoneID = pull_random_element([stoneIDs], lambda id: matcher(gossipLocations[id].name))
             if stoneID is None:
-                raise RuntimeError('Gossip stone unknown or already assigned in world %d: %r. %s' % (self.id + 1, name, build_close_match(name, 'stone')))
+                # Allow planning of explicit textids
+                match = re.match('^\$([a-fA-F0-9]{4})$', name)
```

Comment:
This could potentially be checked before the pattern_matcher.
I wanted to suggest integrating it with the pattern_matcher to get the ability to pick a random from a list but we don't list all the arbitrary IDs in stoneIDs, so I guess this will be fine.